WhaleSpotting/react-app/src/styles/Footer.scss

```diff
@@ -0,0 +1,61 @@
+@import "./Constants.scss";
+
+.footer {
+    position: fixed;
+    bottom: 0px;
+    background-color: $primaryDark;
+    display: flex;
+    justify-content: center;
+    width: 100vw;
+    height: $footerHeight;
+    min-height: 70px;
+
+    .footer-logo-container {
+        max-width: $bodyWidth;
+        width: 85%;
+        display: flex;
+        justify-content: space-between;
+
+        .link {
+            padding: 1%;
+            color: $textOnPrimary;
+            font-family: $fontFamily;
+            text-decoration: none;
+
+            .logo {
+                height: 100%;
+            }
+
+            &.whale-link {
+                margin-left: 10%;
```

Comment:
I think I prefer this without the left margin tbh, but happy to go with what you think if you prefer :)